scoring.js

```diff
@@ -0,0 +1,63 @@
+//Scoring Module
+
+// The lower score the better
+// Points are given for each mile between the coordinate selected and the coordinate from the flickr image
+// you can also get points for taking the hint
+
+/**
+* Haversine formula for distance between coordinates, removed the non-miles calculations
+*/
+function distance(lat1, lon1, lat2, lon2) {
```

Comment:
I've got latitude and longitude packed into a single object {lat: 45.5, lng: -122} because we can't return tuples.

I guess it doesn't matter much in this space?